server/src/test/proto/spine/test/procman/exam/commands.proto

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright 2018, TeamDev Ltd. All rights reserved.
+//
+// Redistribution and use in source and/or binary forms, with or without
+// modification, must retain the above copyright notice and the following
+// disclaimer.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+syntax = "proto3";
+
+package spine.test.procman.exam.command;
+
+import "spine/options.proto";
+import "spine/server/entity/entity.proto";
+
+option (type_url_prefix) = "type.spine.io";
+option java_package = "io.spine.test.procman.exam.command";
+option java_outer_classname = "ProcmanExamCommandsProto";
+option java_multiple_files = true;
+
+import "spine/test/procman/exam/exam.proto";
+
+message PmStartExam {
+    PmExamId exam_id = 1;
+    repeated PmProblemId problem = 2;
```

Comment:
It looks weird. Starting an exam assumes that it already exists.

I'd say `CreateTest`, `StartTest`, `ProposeSolution` etc. To me that's more common.